cpu/native/socket_zep/socket_zep.c

```diff
@@ -390,6 +393,19 @@ static int _connect_remote(socket_zep_t *dev, const socket_zep_params_t *params)
     return res;
 }
 
+static void _send_zep_hello(socket_zep_t *dev)
+{
+    /* dummy packet */
+    zep_v2_data_hdr_t hdr = {
+        .hdr.preamble = "EX",
+        .hdr.version  = 2,
+        .type = SOCKET_ZEP_V2_TYPE_HELLO,
+        .resv = "HELLO",
+    };
+
+    real_write(dev->sock_fd, &hdr, sizeof(hdr));
```

Comment:
Why use `real_write` and not `real_send` here?

Reply:
Why use `real_send` instead of `real_write`?

Reply:
Because you are _sending_ a UDP packet...?

Reply:
But I'm *writing* to a socket

> With a zero flags argument, `send()` is equivalent to `write(2)`.

I can change it if you insist.

Reply:
No, just wanted to know your reasoning.